open-metadata-resources/open-metadata-deployment/charts/vdc/values.yaml

```diff
@@ -55,71 +55,61 @@ debug:
 logging:
   level: NONE
 
-
-# The following section defines all of the DOCKER images being used by this
-# chart. Normally they should be left as is, but are exposed here in case the
-# user wishes to extend.
+imageDefaults:
+  registry: "localhost:5000"
```

Comment:
I am considering reverting the default registry to dockerhub (and making the repository configurable).
I am still awaiting the LF build team, but in the meantime I've had a couple of people ask about using the charts.
The easiest way of doing that is to ensure the images work 'out of the box' MUCH though I dislike the non-official repo id which I will again try and push forward. For them to build, and setup a repo, is not as easy - though it is important to have the capability

I can do this under an additional PR if easier - we don't need to change this one /miss out on the other changes

Reply:
Raised #829